### PR TITLE
Push to elastic

### PR DESCRIPTION
**Pull request by KudumulaGangaNandini**

### CODE REVIEW

File: src/main/java/com/techsophy/tsf/form/service/impl/Status.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.techsophy.tsf.form.service.impl;
+
+
+public enum Status {
+    ENABLED,DISABLED;
```
Comment (VaibhavTechsophy) [inline src/main/java/com/techsophy/tsf/form/service/impl/Status.java]: pls use small case

File: src/test/java/com/techsophy/tsf/form/service/FormsServiceTest.java [hunk 2]
```diff
@@ -148,12 +182,27 @@ void updateFormTypFormTest() throws IOException
...
+        formSchemaTest.setType(TYPE_FORM);
+        formSchemaTest.setVersion(VERSION_VALUE);
+        formSchemaTest.setIsDefault(IS_DEFAULT_VALUE);
+        formSchemaTest.setElasticPush(elasticPush);
+        FormAuditSchema formAuditSchema =new FormAuditSchema();
+        formAuditSchema.setId(ID_VALUE);
+        formAuditSchema.setName(NAME);
+        formAuditSchema.setComponents(COMPONENTS);
+        formAuditSchema.setAcls(List.of(accessControlListDTO));
+        formAuditSchema.setElasticPush(elasticPush);
```
Comment (VaibhavTechsophy) [inline src/test/java/com/techsophy/tsf/form/service/FormsServiceTest.java:205]: is this really required to be set?

File: src/test/java/com/techsophy/tsf/form/service/FormsServiceTest.java [hunk 3]
```diff
@@ -126,13 +149,24 @@ void saveFormAndSaveTest() throws IOException
...
+        formSchemaTest.setName(NAME);
+        formSchemaTest.setComponents(COMPONENTS);
+        formSchemaTest.setAcls(List.of(accessControlListDTO));
+        formSchemaTest.setElasticPush(elasticPush);
+        FormAuditSchema formAuditSchema =new FormAuditSchema();
+        formAuditSchema.setId(ID_VALUE);
+        formAuditSchema.setName(NAME);
+        formAuditSchema.setComponents(COMPONENTS);
+        formAuditSchema.setAcls(List.of(accessControlListDTO));
+        formAuditSchema.setElasticPush(elasticPush);
```
Comment (VaibhavTechsophy) [inline src/test/java/com/techsophy/tsf/form/service/FormsServiceTest.java:169]: is this really required to be set?

**Review by VaibhavTechsophy — CHANGES_REQUESTED** [review]
Please add lint and test coverage count.

File: src/main/java/com/techsophy/tsf/form/dto/FormResponseSchema.java [hunk 4]
```diff
@@ -11,8 +12,8 @@
 import static com.techsophy.tsf.form.constants.FormModelerConstants.*;
 import static com.techsophy.tsf.form.constants.FormModelerConstants.REGEX_CONSTANT;
 
-@With
-@Value
+@Data
+@AllArgsConstructor
```
Comment (VaibhavTechsophy) [inline src/main/java/com/techsophy/tsf/form/dto/FormResponseSchema.java]: can we avoid using all args constructor

File: src/test/java/com/techsophy/tsf/form/controller/FormAuditControllerImplTest.java [hunk 5]
```diff
@@ -113,8 +114,9 @@ void saveFormComponentTypeTest() throws Exception
     @Test
     void getFormByIdTest() throws Exception
     {
+        Status status = Status.DISABLED;
         AccessControlListDTO accessControlListDTO = new AccessControlListDTO(TYPE,"value",true,true,true,true,true);
-        FormAuditResponseSchema formSchemaTest =new FormAuditResponseSchema(ID_VALUE,FORM_ID_VALUE,NAME,COMPONENTS, List.of(accessControlListDTO),PROPERTIES,TYPE_FORM, VERSION_VALUE, IS_DEFAULT_VALUE ,CREATED_BY_ID_VALUE, CREATED_ON_INSTANT, UPDATED_BY_ID_VALUE, UPDATED_ON_INSTANT);
+        FormAuditResponseSchema formSchemaTest =new FormAuditResponseSchema(ID_VALUE,FORM_ID_VALUE,NAME,COMPONENTS, List.of(accessControlListDTO),PROPERTIES,TYPE_FORM, VERSION_VALUE, IS_DEFAULT_VALUE ,CREATED_BY_ID_VALUE, CREATED_ON_INSTANT, UPDATED_BY_ID_VALUE, UPDATED_ON_INSTANT,status);
```
Comment (VaibhavTechsophy) [inline src/test/java/com/techsophy/tsf/form/controller/FormAuditControllerImplTest.java]: please avoid using these constructors

File: src/test/java/com/techsophy/tsf/form/utils/TokenUtilsTest.java [hunk 6]
```diff
@@ -146,6 +147,20 @@ void getPaginationResponsePayload()
         PaginationResponsePayload responsePayload = tokenUtils.getPaginationResponsePayload(page,list);
         assertThat(responsePayload).isNotNull();
     }
+    @Test
+    void getTenantTest()
+    {
+        Authentication authentication = Mockito.mock(Authentication.class);
```
Comment (VaibhavTechsophy) [inline src/test/java/com/techsophy/tsf/form/utils/TokenUtilsTest.java:153]: why these test cases are modified as part of this implementation

**Review by VaibhavTechsophy — CHANGES_REQUESTED** [review]

File: src/test/java/com/techsophy/tsf/form/utils/TokenUtilsTest.java (hunk 6 above)
Comment (KudumulaGangaNandini) [reply src/test/java/com/techsophy/tsf/form/utils/TokenUtilsTest.java:153]: in the process of converting from All args constructor to set method ,  code coverage is decreased sir.so i cover with this method

**Review by VaibhavTechsophy — APPROVED** [review]

**Pull request merged**